index.html

```diff
@@ -116,7 +116,33 @@ <h2 class="about-title">About me</h2>
 <p class="about-contend">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
 <a href="#">Get My Resume</a>
 <div class="ornament2" ></div>
-
 </section>
+
+
+
+
+<footer class="form">
+<form class="form" action="https://formspree.io/f/xaykaovn" method="post">
+<p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+<input class="form-name" placeholder="Full name" type="text" name="form_name" id="" required>
+<input class="form-email" placeholder="Email address" type="email" name="form_email" id="" required >
+<!-- <input class="form-coment" placeholder="Enter text here" type="text"  maxlength="500" name="" id=""> -->
+<textarea class="form-coment" placeholder="Enter text here" name="form_coment" id="" cols="30" rows="10" required></textarea>
+<button type="submit" >Get in touch</button>
+</form>
+
+
+<ul class="s-media">
+  <li class="cat media"><img src="pictures/cat.svg" alt=""></li>
+  <li class="in media"><img src="pictures/linkedin.svg" alt=""></li>
+  <li class="v media"><img src="pictures/victori.svg" alt=""> </li>
+  <li class="twitter media"><img src="pictures/twitter2.svg" alt=""></li>
+  <li class="m media"><img src="pictures/M.svg" alt=""></li>
```

Comment:
- [ ] please ensure the social icons have the figma color.
- [ ] also, add the necessary padding and margin to the submit button
- [ ] Also kindly add the the line under the social icon so it follows the figma design
yours

![Screenshot (579)_LI](https://user-images.githubusercontent.com/65298048/200874707-bc499c3e-ac27-41ee-ad41-44b9143a9aa8.jpg)

figma
![Screenshot (580)](https://user-images.githubusercontent.com/65298048/200874905-52534d30-14df-4c5d-9f73-cfe845c25458.png)